### ISSUE TITLE
Add Connection Icons

### ISSUE THREAD

**Issue by kahboom** [redhat-ipaas/ipaas-ui#421]
## Overview
Add "connector" icons in the UI, reference the path using the connector name. See full conversation [here](https://github.com/redhat-ipaas/ipaas-rest/issues/293).

## Reference
- [JIRA Issue: UI](https://issues.jboss.org/browse/IPAAS-295)
- [REST Issue](https://github.com/redhat-ipaas/ipaas-rest/issues/293)

**Comment by kahboom** [comment]
This is completed for Connections, including for Connectors in Create a Connection, the Dashboard, and Connection List. Need to do the same for integrations, but was having an issue accessing the following property, got `undefined` on the `steps` property: `integration.steps[0].connection.connectionId`

This should be an empty array if anything, not undefined. The error occurs even when I add a check for `integration` first. Will check again now in case my brain was just fried last night.

cc @gashcrumb @KurtStam

**Comment by gashcrumb** [comment]
Should be `connectorId` not `connectionId` unless that's just a typo :-)

**Comment by gashcrumb** [comment]
@kahboom just checking the network tab for the response back from listing integrations, looks like the data is as I'd expect, i.e. `steps[0].connection.connectorId` on 'Timed Pull to Post Example` is set to `timer`.

**Comment by kahboom** [comment]
@gashcrumb Yep that was a typo, but let me try again with `connectorId`..

**Comment by kahboom** [comment]
Waiting on the remaining icons specified in the comments [here](https://issues.jboss.org/browse/IPAAS-295).

**Comment by kahboom** [comment]
And, of course, to add icons for integrations. :)

**Comment by jimmidyson** [comment]
Fixed by #446 hopefully.
